Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 authors = ["Sean Bowe <ewillbefull@gmail.com>, CPerezz <carlos@dusk.network>"]
 description = "Fork of the implementation of the BLS12-381 pairing-friendly elliptic curve construction with some extra tooling needed by the Dusk team"
-documentation = "https://github.com/dusk-network/bls12_381"
+documentation = "https://docs.rs/dusk-bls12_381"
```

Comment:
You can omit the documentation if it lands on the default doc.rs 

For example, look at kadcast crates and its cargo.toml

Reply:
> You can omit the documentation if it lands on the default doc.rs
> 
> For example, look at kadcast crates and its cargo.toml

I'm aware of this, but decided to make it explicit anyway given that there was a value already defined. I'll delete it anyway.